src/features/farming/crops/components/GoblinShovel.tsx

```diff
@@ -87,6 +88,9 @@ export const GoblinShovel: React.FC = () => {
   };
 
   const shovelImage = state.inventory.Shovel?.gte(1) ? shovel : rustyShovel;
+  const goblinstealerState = state.inventory.Shovel?.gte(1)
```

Comment:
Suggestion: leave this just as goblin to not get too wordy and add a comment:
`//if the normal shovel hasn't been crafted show rusty instead of normal shovel`
  `const shovelImage = state.inventory.Shovel?.gte(1) ? shovel : rustyShovel;`
  `const goblin = state.inventory.Shovel?.gte(1) ? shovelGoblin : rustyShovelGoblin;`